todo_app/app.py

```diff
@@ -42,43 +39,33 @@ def add_todo():
 
             to_add = {
                 'name': name,
+                'modified_date': datetime.now().isoformat(),
+                'is_done': False
             }
 
             if description.strip() != '':
-                to_add['desc'] = description
+                to_add['description'] = description
```

Comment:
Minor, but worth noting that having these lines here change suggests that this code "cares" about the fact our persistence has changed, even though the user experience should be pretty much identical. This is very mild given these might be nicer names anyway, but worth considering that if we'd left the names as is, we could easily have used the `MongoItem.from_dict` method to handle any changing field names under the hood, so that this code barely needs to change at all